### PR TITLE
Copy and move page

### PR DESCRIPTION
**Pull request by emcelroy**
PT Story: https://www.pivotaltracker.com/story/show/178595321

[#178595321]

### CODE REVIEW

**Review by knowuh — APPROVED** [review]
This looks good Ethan. Thank you!

I think we should merge it.

File: lara-typescript/src/page-nav-menu/components/icons/copy-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+
+const kDefaultHight = "1em";
+const kDefaultWidth = "1em";
+
+export interface ICopyProps {
+  fillColor?: string;
```
[the remark this answers is not in the document]
Comment (emcelroy) [reply lara-typescript/src/page-nav-menu/components/icons/copy-icon.tsx]: Yes, looks like I forgot to remove it. Thanks for catching that. I've removed it in the latest commit.

[the remark this answers is not in the document]
Comment (knowuh) [reply lara-typescript/src/page-nav-menu/components/icons/copy-icon.tsx]: I think all the SVG components have that same prop.  Not sure if any of them use it.

[the remark this answers is not in the document]
Comment (emcelroy) [reply lara-typescript/src/page-nav-menu/components/icons/copy-icon.tsx]: Ah, you're right. Pretty much all of them did. None of them use it, though. I've made a more thorough purging of that parameter.

**Review by knowuh — COMMENTED** [review]
Looks good, thanks

**Review by knowuh — APPROVED** [review]

**Pull request merged**